tests/test_json_schema.py

```diff
@@ -5947,3 +5932,23 @@ class Model(BaseModel):
         c: ModelC
 
     assert Model.model_json_schema()
+
+
+def test_json_schema_annotated_with_field() -> None:
+    """Check if the ellipsis in the signature is considered as a required field."""
+
+    from pydantic import create_model
+
+    Model = create_model(
+        'test_model',
+        bar=(Annotated[int, Field(description='Bar description')], ...),
```

Comment:
Could you include this example as well? `baz=(Annotated[int, Field(..., description="Baz description")], ...),`

Reply:
Maybe with this model:

```py
Model = create_model(
    'test_model',
    foo=(int, ...),
    bar=(Annotated[int, Field(description="Bar description")], ...),
    baz=(Annotated[int, Field(..., description="Baz description")], ...),
)
```

Reply:
Done 😄